Sources/SwiftCurrent_SwiftUI/Views/WorkflowLauncher.swift

```diff
@@ -21,31 +21,27 @@ import SwiftCurrent
 /// WorkflowLauncher(isLaunched: $isLaunched.animation(), args: "String in")
 ///     .thenProceed(with: WorkflowItem(FirstView.self)
 ///                     .applyModifiers {
-///         if true { // Enabling transition animation
 ///             $0.background(Color.gray)
 ///                 .transition(.slide)
 ///                 .animation(.spring())
-///         }
-///     })
+///     }
 ///     .thenProceed(with: WorkflowItem(SecondView.self)
 ///                     .persistence(.removedAfterProceeding)
 ///                     .applyModifiers {
-///         if true {
 ///             $0.SecondViewSpecificModifier()
 ///                 .padding(10)
 ///                 .background(Color.purple)
 ///                 .transition(.opacity)
 ///                 .animation(.easeInOut)
-///         }
-///     })
+///     }))
 ///     .onAbandon { print("isLaunched is now false") }
 ///     .onFinish { args in print("Finished 1: \(args)") }
 ///     .onFinish { print("Finished 2: \($0)") }
 ///     .background(Color.green)
 ///  ```
 @available(iOS 14.0, macOS 11, tvOS 14.0, watchOS 7.0, *)
 public struct WorkflowLauncher<Args> {
-    @Binding private var isLaunched: Bool
+    @Binding var isLaunched: Bool
```

Comment:
For testability?  I don't remember this change.

Reply:
No, for `WorkflowLauncherView`

Reply:
I'm must be dumb, because I can't see it in `WorkflowLauncherView`

Reply:
Line 40

Reply:
There we go, I see it now. Thank you!